lcUI/cadmdichild.cpp

```diff
@@ -16,7 +16,8 @@ using namespace lc::viewer;
 CadMdiChild::CadMdiChild(QWidget* parent) :
     QWidget(parent),
     _id(0),
-    _activeLayer(nullptr) {
```

Comment:
Please avoid those kind of changes. The file wasn't changed but 27 lines appear to be changed because of spaces...
You can reset this file from origin/master